### PR TITLE
Scroll when you send a DM

### PR DESCRIPTION
**Pull request by Connoropolous**
regardless of whether you've scrolled up/back through the conversation or not

### CODE REVIEW

File: src/components/MessageSection.js [hunk 1]
```diff
@@ -36,8 +36,11 @@ export default class MessageSection extends React.Component {
   componentDidUpdate (prevProps) {
     const messagesLength = this.props.messages.length
     const oldMessagesLength = prevProps.messages.length
+    const { currentUser } = this.context
+    const latestMessage = reverse(sortBy(this.props.messages || [], 'created_at'))[0]
```
Comment (levity) [inline src/components/MessageSection.js]: rather than reversing the entire array to get the last item, let's just do `list[list.length - 1]`

Comment (levity) [reply src/components/MessageSection.js]: or `maxBy('created_at')`, which makes it a one-step process rather than having to sort then look up

**Review by levity — CHANGES_REQUESTED** [review]
make the suggested change to `latestMessage` and remove the `.swp` file that was checked in, & we're good

File: src/components/MessageSection.js (hunk 1 above)
Comment (Connoropolous) [reply src/components/MessageSection.js]: K!

**Comment by Connoropolous** [comment]
@levity changes made

**Pull request merged**